includes/tracker.php

```diff
@@ -337,6 +341,124 @@ public static function get_library_usage() {
 
 	}
 
+	/**
+	 * Get usage of general settings.
+	 * 'Elementor->Settings->General'.
+	 *
+	 * @return array
+	 */
+	public static function get_settings_general_usage() {
+		$result = [];
+
+		// Posts types.
+		$cpt_support = get_option( 'elementor_cpt_support', [] );
+		$cpt_usage = [];
+
+		foreach ( Plugin::ELEMENTOR_POST_TYPES as $post_type ) {
+			$cpt_usage[ $post_type ] = in_array( $post_type, $cpt_support );
+		}
```

Comment:
Did I miss somthing?
```suggestion
		$cpt_support = get_option( 'elementor_cpt_support', Plugin::ELEMENTOR_POST_TYPES );
```

Reply:
Handled.